packages/rest/.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Not sure if it is possible but we should have a prettierrc for the entire monorepo, not each package.